.ci_support/linux_64_c_compiler_version10c_stdlib_version2.17cuda_compilernvcccuda_compiler_version11.2cxx_compiler_version10python3.10.____cpython.yaml

```diff
@@ -0,0 +1,42 @@
+c_compiler:
```

Comment:
Accidently left these in when merging changes for PyTorch 2.3 support, can remove these.